### DISCUSSION TITLE
Addition of a Difficulty Card Anchor for editor components

### DISCUSSION THREAD

**Discussion by iMaestr0i** [ppy/osu#25453] [Ideas]
Just came up with a few things I think would be neat features.

First, the Diff Card anchor.

I was messing around with the skin editor, and ended up with this
![StarText2](https://github.com/ppy/osu/assets/150843238/24c61238-6c80-4754-8aba-0be43e94c1a3)

This would be nice to have for a couple of reasons.

1. It's currently up by the song card (which I often forget to check before starting a map.)

2.  It's a bit easier to see at a glance when picking difficulty.

3.  i dunno it just looks slick lol

One small issue.
The moment I scroll, it causes issues.

![osu_2023-11-14_12-57-47](https://github.com/ppy/osu/assets/150843238/0dbbdd84-a509-4587-8acf-4ef7fc97900e)
![osu_2023-11-14_12-57-56](https://github.com/ppy/osu/assets/150843238/dabc0e41-e87d-441b-af32-d98af9a0aa8d)
![osu_2023-11-14_12-58-09](https://github.com/ppy/osu/assets/150843238/98cf3fd3-c39b-4a88-9f1a-5eb2ea806b00)

So my thought process is if there were a way to add these elements into the actual card itself, this wouldn't be a problem.
I'm not sure how technically involved implementing a feature like this would be, but I think it would be great!

**Comment by Joehuu** [comment]
The incoming new design has star rating on those panels by default (still need to consider how skinnable these components are), see https://github.com/ppy/osu/issues/22204 for the full song select redesign progress (see figma designs).

I was working on it and will revisit and hopefully get the changes into a reviewable state before this month ends.
